Pinta.Effects/Utilities/EffectHelper.cs

```diff
@@ -34,19 +34,15 @@ internal static class EffectHelper
 	/// <summary>
 	/// Launch an effect dialog using Pinta's translation template.
 	/// </summary>
-	internal static void LaunchSimpleEffectDialog (BaseEffect effect)
-	{
-		PintaCore.Chrome.LaunchSimpleEffectDialog (effect, new PintaLocalizer ());
-	}
+	internal static void LaunchSimpleEffectDialog (this IChromeManager chrome, BaseEffect effect)
```

Comment:
Is this function unused now?

Reply:
Not sure what you mean. I didn't delete it, but just changed its signature

Reply:
Yeah I think I was mis-reading the diff, sorry